### ISSUE TITLE
Bad audio playing songs from youtube

### ISSUE THREAD

**Issue by s-hell** [mopidy/mopidy-youtube#13]
Hi,
i'm having a crackling sound listen to youtube-videos. Playing the same video in flash, html5 or with mplayer over the same internet connections, the sound ist good.
May be it's some caching problem?!?

**Comment by dz0ny** [comment]
99% hardware related or at least linux subsystem. RPi?

**Comment by s-hell** [comment]
I have install mopidy on my laptop and on a raspberry pi. Both have the same problem.
Normal Audio Streams work fine.

**Comment by brunorey** [comment]
Mopidy uses gstreamer (gst). Can you play youtube streams with gstreamer?

Links:
https://docs.mopidy.com/en/latest/config/#advanced-configurations
http://askubuntu.com/questions/526258/how-to-add-audio-elements-and-video-elements-for-gstreamer-with-qtdemux
http://lists.freedesktop.org/archives/gstreamer-bugs/2011-August/081082.html

**Comment by dz0ny** [comment]
You can confirm that this is not a mopidy issue by directly using gst to play youtube track:
```
gst-launch-0.10 playbin2 uri=(echo "https://www.youtube.com/watch?v=RuY4MSh1V-I" | python2 -c 'import pafy,sys;print(pafy.new("".join(sys.stdin)).getbestaudio().url)')
```

**Comment by dz0ny** [comment]
@brunorey ping

**Comment by brunorey** [comment]
@dz0ny , I believe you were trying to ping @s-hell who originally had the issue. I was just trying to help in debugging :-)

**Comment by simsasaile** [comment]
Hi, I have the same problem :( 

Using gstreamer directly as mentioned [here](https://github.com/dz0ny/mopidy-youtube/issues/13#issuecomment-75289811) is working fine.

**Comment by simsasaile** [comment]
The problem seems to be connected with this topic: https://github.com/mopidy/mopidy/issues/1147
I increased the value as described there and now the quality is really nice!

**Comment by jodal** [comment]
Ref the last comment, this should have been fixed with the release of Mopidy 1.0.4. Bad releases include Mopidy 1.0.0 through 1.0.3.
